gen_dockerfiles

```diff
@@ -132,7 +131,18 @@ RUN { \\
 	echo 'deb http://neuro.debian.net/debian $release main'; \\
 	echo 'deb http://neuro.debian.net/debian data main'; \\
 	echo '#deb-src http://neuro.debian.net/debian-devel $release main'; \\
-} > /etc/apt/sources.list.d/neurodebian.sources.list
+} > /etc/apt/sources.list.d/neurodebian.sources.list \\
+    && apt-get update
+
+# Minimalistic package to assist with freezing the APT configuration
+# which would be coming from neurodebian repo.
+# Also install and enable eatmydata to be used for all apt-get calls
+# to speed up docker builds.
+RUN apt-get install -y --no-install-recommends neurodebian-freeze eatmydata \\
+    && ln -s /usr/bin/eatmydata /usr/local/bin/apt-get
+
+# Clean up
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
This `rm` needs to be on each layer (`RUN`) that does an `apt-get update`, otherwise you won't save any space by deleting the apt lists since it is already part of an earlier docker layer.

Reply:
Thank you! I will fix for that